Assets/MRTK/Examples/Demos/HandTracking/Profiles/HandMenuExamplesInputSimulationProfile.asset

```diff
@@ -0,0 +1,68 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Do we need this profile, and the other ones below it?

Reply:
I believe we just need this one; I don't think we actually need the pointer profile (though it appears that that's the preferred way to activate eye gaze, so it might be beneficial to keep it for now)

Reply:
Per above comment, I think we can remove it and behavior will be same. Same with all the profiles below.